spin/cmds/meson.py

```diff
@@ -26,6 +26,32 @@ def _meson_cli():
         return [meson_cli]
 
 
+def _editable_install_path(package):
+    import importlib_metadata
+
+    try:
+        dist = importlib_metadata.Distribution.from_name(package)
```

Comment:
I think `Distribution.from_name(...)` expects the distribution name and not the package name! 

As is, this line always raises a `PackageNotFoundError` despite the editable install clearly working otherwise (installed with `pip install --no-build-isolation -e .`). Running `spin test ...` etc. still builds as if the editable install is not present.

However, when I set a breakpoint here and use 
```
importlib_metadata.Distribution.from_name("scikit-image")
``` 
it seems to work! Do you have access to the distribution name at this level?

Reply:
I'm surprised this is working for you? Maybe my setup is weird?